docs/source/index.rst

```diff
@@ -2,54 +2,101 @@
 Dask
 ====
 
-Dask is a simple task scheduling system that uses directed acyclic graphs (DAGs)
-of tasks to break up large computations into many small ones.
+Dask is a flexible parallel computing library for analytics.  Dask emphasizes
+the following virtues:
 
-**Dask enables parallel computing** through task scheduling and blocked algorithms.
-This allows developers to write complex parallel algorithms and execute them
-in parallel either on a modern multi-core machine or on a distributed cluster.
+*  **Familiar**: Provides parallelized NumPy array and Pandas DataFrame objects
```

Comment:
It's good to mention Dask dataframes and arrays here, is there somewhere else in this topmost list to refer to bags and imperative?